packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.tsx

```diff
@@ -100,6 +104,18 @@ export const VerticalBarChart: React.FunctionComponent<IVerticalBarChartProps> =
   const [callOutAccessibilityData, setCalloutAccessibilityData] = React.useState<IAccessibilityProps>();
   const [dataPointCalloutProps, setDataPointCalloutProps] = React.useState<IVerticalBarChartDataPoint>();
 
+  function _uniqueLineId(): string {
+    return `${_vbcLineId}-${vbcLineCounter++}`;
+  }
+
+  function _uniquePointId(): string {
+    return `${_vbcPointId}-${vbcPointCounter++}`;
+  }
+
+  function _uniqueBarId(): string {
+    return `${_vbcBarId}-${vbcBarCounter++}`;
```

Comment:
use the index directly. Dont update the id on each rerender

Reply:
pushed changes